meson_options.txt

```diff
@@ -40,5 +40,5 @@ option('enable-openmp', type: 'boolean', value: true)
 # If this is disabled, related options (capi-ml-*) are ignored.
 option('ml-api-support', type: 'feature', value: 'auto')
 # @todo : make them use 'feature' and depend on ml-api-support
-option('enable-nnstreamer-tensor-filter', type: 'boolean', value: false)
+option('enable-nnstreamer-subplugin', type: 'boolean', value: false)
```

Comment:
please seperate options for filter-subplugin and trainer-subplugin by keeping enable-nnstreamer-tensor-filter and adding enable-nnstreamer-tensor-trainer.

You may use 'feature" instead of 'boolean' by checking the availability for the required nnstreamer subplugin API header.

Reply:
okay thank you.

Reply:
Now enable-nnstreamer-tensor-filter and enable-nnstreamer-tensor-trainer are set by feature insted of boolean.
I added define for header file.